setup.cfg

```diff
@@ -57,7 +57,7 @@ classifiers =
 packages = find:
 zip_safe = False
 include_package_data = False
-python_requires = >=3.8, <3.13
+python_requires = >=3.8, <3.14
```

Comment:
Don't forget the `Programming Language :: Python :: 3.12` classifier as well

Reply:
> Don't forget the `Programming Language :: Python :: 3.12` classifier as well

Good catch!